src/content/docs/Artist entry editing.mdx

```diff
@@ -60,7 +60,7 @@ From [source text](source url):
 
 Which displays as:
 
-From [source text](source url):
+From [source text]():

```

Comment:
This line is to highlight how markdown links work.

Maybe `- From [source text](https://example.com/):` would work better.